src/target/source/codegen_cuda.cc

```diff
@@ -46,6 +48,39 @@ void CodeGenCUDA::Init(bool output_ssa) {
 
 void CodeGenCUDA::PrintFuncPrefix() { stream << "extern \"C\" __global__ void"; }
 
+class threadIdxExtractor : public tir::StmtVisitor {
```

Comment:
Wait please fix the naming style

```suggestion
class ThreadIdxExtractor : public tir::StmtVisitor {
```

Reply:
Fixed